meson.build

```diff
@@ -2494,6 +2494,10 @@ meson.add_install_script(meson_make_symlink,
                          rootlibexecdir / 'systemd',
                          rootsbindir / 'init')
 
+meson.add_install_script(meson_make_symlink,
```

Comment:
isn't this best done when building the initrd image?

Reply:
It's relevant even when we're not building an initrd. You might want to boot into a cpio without switching root. In that case you have a cpio that's not an initrd, but you still need the /init symlink for the kernel to do the right thing. Given that we install /sbin/init already, I figure we might as well install this symlink as well.

Reply:
Hmm, dropped this commit since it might interfere with ramdisk loading.